Shared/UI/Face Detection/FaceImageCaptureViewController.swift

```diff
@@ -22,6 +22,7 @@ class FaceImageCaptureViewController: ViewController {
     enum VideoCaptureState {
         case idle
         case starting
+        case started
```

Comment:
We need wait for the video input to be ready, otherwise the video has black frames at the beginning. 

Adding this state let's us mark that we're ready to process frames.